e2e_test/source/cdc_inline/sql_server_cdc/sql_server_cdc.slt

```diff
@@ -0,0 +1,259 @@
+
+control substitution on
+
+# ------------ data prepare stage ------------
+system ok
+sqlcmd -d master -Q 'create database mydb;' -b
+
+system ok
+sqlcmd -C -i e2e_test/source/cdc_inline/sql_server_cdc/sql_server_cdc_prepare.sql -b
+
+# ------------ validate stage ------------
+# TODO(kexiang): add more tests here
+# invalid username
+# invalid password
+# invalid table name
+# invalid primary key
+# column name mismatch
+# column data type mismatch
+# format & encode provided and match with debezium json, this is okay
+# format & encode provided but mismatch with debezium json, this is not allowed
+
+
+statement error
+CREATE TABLE orders (
+  order_id INT PRIMARY KEY,
+  order_date BIGINT,
+  customer_name VARCHAR,
+  price DECIMAL,
+  product_id INT,
+  order_status SMALLINT
+) WITH (
+  connector = 'sqlserver-cdc',
+  hostname = '${SQLCMDSERVER:sqlserver-server}',
+  port = '${SQLCMDPORT:1433}',
+  username = '${SQLCMDUSER:SA}',
+  password = '${SQLCMDPASSWORD}',
+  table.name = 'orders',
+  database.name = '${SQLCMDDBNAME}',
+);
+
+statement error
```

Comment:
suggest to add error messages in the end.

Reply:
done